### DISCUSSION TITLE
Priority settings for SPH3600 (Batt First, Load First, Grid First)

### DISCUSSION THREAD

**Discussion by Rasmusbangp** [wills106/homeassistant-solax-modbus#1084] [Q&A]
Hello 

We are trying to understand the behavior of the SPH3600(SPH3000-6000) 

For MOD, SPH10k and others we know they the sequences of setting the priority is to set the time slot at the same time as the desired priority is enabled.

But some is different at the SPH3000-6000- 

can someone please assist how do we set this, certainly the GRID FIRST cannot be activated 🧐

**Comment by OliverLang123** [comment]
Any update on this? Did anyone figure out how to properly set priorities on SPH3000–6000 (GRID FIRST in particular)?
